bcda/cclf/cclf.go

```diff
@@ -266,6 +266,8 @@ func importCCLF(ctx context.Context, fileMetadata *cclfFileMetadata, importer im
 		}
 	}()
 
+	var importedMBI = make(map[string]models.CCLFBeneficiary)
```

Comment:
Since we only need to track the existence of the MBI, we should use a `map[string]struct{}`. This effectively allows us to use a map like a set.